dataactcore/interfaces/function_bag.py

```diff
@@ -17,15 +19,16 @@
 
 def createUserWithPassword(email, password, bcrypt, permission=1, cgac_code="SYS"):
     """Convenience function to set up fully-baked user (used for setup/testing only)."""
-    sess = GlobalDB.db().session
-    status = sess.query(UserStatus).filter(UserStatus.name == 'approved').one()
-    user = User(email=email, user_status=status, permissions=permission,
-                cgac_code=cgac_code)
-    pwd = getPasswordHash(password, bcrypt)
-    user.salt = pwd[0]
-    user.password_hash = pwd[1]
-    sess.add(user)
-    sess.commit()
+    with createApp().app_context():
```

Comment:
We probably want to create this context outside of `createUserWithPassword` -- to make it as unobtrusive as possible, we'd want to push the context set up as close to the "start" of the application as possible. Maybe at the test setup?